components/wrist.py

```diff
@@ -59,11 +61,7 @@ def __init__(self, mech_root: wpilib.MechanismRoot2d):
             (1 / 60) * math.tau * (1 / self.wrist_gear_ratio)
         )
 
-        self.motor.configure(
-            wrist_config,
-            SparkMax.ResetMode.kResetSafeParameters,
-            SparkMax.PersistMode.kPersistParameters,
-        )
+        configure_spark_ephemeral(self.motor, wrist_config)
```

Comment:
Do we not want to continue persisting these configs at robot start?

Reply:
Done the thing